api/tests/integration/AuthTest.php

```diff
@@ -569,7 +530,7 @@ public function testSingleSignOnVanillaWithUser()
         $token = $this->tokenFromUser($user);
 
         $params = ['client_id' => env('VANILLA_JSCONNECT_CLIENT_ID')];
-        $cookies = [\App\Http\Controllers\AuthController::JWT_AUTH_TOKEN_COOKIE => $token];
+        $cookies = ['ngJwtAuthToken' => $token];
```

Comment:
Use the constant here, as we might change the value, and we only want to change it in one place.